bundle/EventListener/LegacyKernelListener.php

```diff
@@ -43,6 +46,20 @@ function() {
 
     public function onConsoleCommand( ConsoleCommandEvent $event )
     {
+        $this->resetKernelHandler();
+
+        $eventDispatcher = $this->container->get( 'event_dispatcher' );
+        $eventDispatcher->addListener(MVCEvents::CONFIG_SCOPE_CHANGE, [$this, 'onConsoleConfigScopeChange'], -1);
+        $eventDispatcher->addListener(MVCEvents::CONFIG_SCOPE_RESTORE, [$this, 'onConsoleConfigScopeChange'], -1);
```

Comment:
Why adding those manually here? AFAICS, you could add those in `getSubscribedEvents()` right?